packages/keymaps/src/common/user-storage-uri.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import URI from "@theia/core/lib/common/uri";
+
+export namespace UserStorageUri {
+
+    export const SCHEME = 'user';
+
+    /**
+     * Creates a new file URI from the filesystem path argument.
+     * @param fsPath the filesystem path.
+     */
+    export function create(fsPath: string) {
+        const fsUri = new URI(fsPath);
+        return new URI('').withScheme(SCHEME).withPath(fsUri.displayName);
```

Comment:
Why `fsUri.displayName`?

Reply:
Should it be fsUri.path.base? If I use fsUri.path.toString() I end up with user_storage:/home/user/.theia/.keymaps.json. Is there an easy way to strip the relative path from theia user dir (i.e not just the filename)? This way */home/user/.theia*/folder/keymaps.json we could extract */folder/keymaps.json*